cgmes/cgmes-conformity/src/test/java/com/powsybl/cgmes/conformity/test/CgmesConformity1NetworkCatalog.java

```diff
@@ -177,8 +177,10 @@ public Network microBE(String modelId) {
                 .setName("BE_S2")
                 .setConnectableBus(busBrussels380.getId())
                 .setBus(busBrussels380.getId())
-                .setbPerSection(3.46e-4)
-                .setMaximumSectionCount(1)
+                .newShuntCompensatorLinearModel()
```

Comment:
I would name it .newLinearModel() because we already known it belongs to a shunt compensator

Reply:
Done.